cirq-core/cirq/ops/gateset_test.py

```diff
@@ -81,6 +81,24 @@ def test_gate_family_default_name_and_description(gate, tags_to_accept, tags_to_
     assert (ignored_match is None) == (tags_to_ignore == [])
 
 
+@pytest.mark.parametrize(
+    'tags_to_accept_fam1, tags_to_ignore_fam1, tags_to_accept_fam2, tags_to_ignore_fam2',
+    [
+        (frozenset("abcd"), frozenset("efgh"), frozenset("dcba"), frozenset("hgfe")),
+        (frozenset("pabdcqwd"), [], frozenset("pcdwdbqa"), []),
+        ([], frozenset("qwerty"), [], frozenset("ytrewq")),
```

Comment:
nit: use `tuple` instead of `frozenset` since the iteration order of frozenset can be arbitrary. Also suggest using smaller strings to make the test cleaner. 

```suggestion
        (tuple("ab"), tuple("cd"), tuple("ba"), tuple("dc")),
        (tuple("ab"), [], tuple("ba"), []),
        ([], tuple("ab"), [], tuple("ba")),
```